include/deal.II/distributed/shared_tria.h

```diff
@@ -151,6 +170,17 @@ namespace parallel
        */
       std::vector<types::subdomain_id> true_subdomain_ids_of_cells;
     };
+
+    template <int dim, int spacedim>
+    template <class Archive>
+    void
+    Triangulation<dim,spacedim>::load (Archive &ar,
+                                       const unsigned int version)
+    {
+      dealii::Triangulation<dim,spacedim>::load (ar, version);
+      partition();
```

Comment:
good point. I think there are two options: load() applied to load the standard `dealii::Triangulation` into `p::s::Tria`. In this case there may be no partitioning. Or load applied to `p::s::Tria`, which should already have `subdomain_id` assigned according to the number of MPI cores. But in this case we would still allow loading `p::s::Tria` used with different number of mpi cores?